.travis.yml

```diff
@@ -19,9 +19,14 @@ cache:
     - $HOME/.cache/pip
 
 python:
-  - "2.7"
-  - "3.5"
   - "3.6"
+  - "3.7"
+  - "3.8"
+
+matrix:
+  allow_failures:
+    - python: 3.7
```

Comment:
modeate: could we try bumping lxml as @tmorrell (Thanks for the hint!) suggests (It's been bumped in other modules in any case). And try to avoid "allowed failures" in 3.7. We want to try to support 3.7 for v3.3 (3.8 is okay in allowed failures)